cmd/mothership/mothership/start/http.go

```diff
@@ -621,16 +621,39 @@ func sendResponse(w http.ResponseWriter, r *http.Request, clusterState *cluster.
 	}
 }
 
-func newClusterResponse(r *http.Request, clusterState *cluster.State) (*keb.HTTPClusterResponse, error) {
+func newClusterResponse(r *http.Request, clusterState *cluster.State, reconciliationRepository reconciliation.Repository) (*keb.HTTPClusterResponse, error) {
 	kebStatus, err := clusterState.Status.GetKEBClusterStatus()
 	if err != nil {
 		return nil, err
 	}
+
+	var failures []keb.Failure
+	reconciliations, err := reconciliationRepository.GetReconciliations(&reconciliation.WithRuntimeID{RuntimeID: clusterState.Cluster.RuntimeID})
```

Comment:
The current implement is unfortunately not fully reliable to retrieve the correct reconciliation for a cluster.

Quick review of the data model:

`Cluster  ---has many-->  ClusterConfiguration ---has many--> ClusterConfigurationStatus`

A reconciliation is always happening on base of a particular `ClusterConfiguration` and not on a runtimeID. A reconciliation entity has following information we can use to query for it:

* reference to the [clusterConfig-ID](https://github.com/kyma-incubator/reconciler/blob/main/configs/db/postgres/000002_scheduler_operations.up.sql#L12)
* reference to the [clusterConfigStatus-ID](https://github.com/kyma-incubator/reconciler/blob/main/configs/db/postgres/000002_scheduler_operations.up.sql#L13)

I would recommend to query for the clusterConfigStatus-ID as it's  a precise match if you have a `cluster.State` instance available.

But this will require to implement quickly another [`Filter`](https://github.com/kyma-incubator/reconciler/blob/main/pkg/scheduler/reconciliation/filter.go) instance which allows querying a reconciliation based on the reference `clusterConfigStatus` ID.